modules/backend/widgets/form/partials/_form_tabs.htm

```diff
@@ -12,9 +12,9 @@
     }
 ?>
 <div class="<?= $navCss ?>">
-    <ul class="nav nav-tabs">
+    <ul class="nav nav-tabs" role="tablist">
         <?php $index = 0; foreach ($tabs as $name => $fields): ?>
-            <li class="<?= $index++==0?'active':''?>">
+            <li class="<?= $index++==0?'active':''?>" aria-controls="<?= $tabs->getIcon($name) ?> Tab" aria-selected="<?= $index++==0?'true':'false'?>" role="tab" tabindex="0">
```

Comment:
I don't think we should have `aria-controls` in here (see https://www.heydonworks.com/article/aria-controls-is-poop) and I also don't think that's how it's implemented even if we did want it.

Also won't the aria-selected be incorrect as soon as they select a different tab?

Reply:
1. I checked out `aria-controls` and they are in the latest spec, see here: https://www.w3.org/TR/wai-aria-1.1/#aria-controls so they are being used.

2. >Also won't the aria-selected be incorrect as soon as they select a different tab?
Going to have a deeper look and play around more with the code. I would think you right.

Reply:
Finished coding please label for testing now